app/views/content_items/_context_and_title.html.erb

```diff
@@ -1,26 +1,7 @@
-<%
-  context_string =  t("content_item.schema_name.#{@content_item.document_type}", count: 1);
-  context_inside = false;
- %>
-
-<% @content_item&.featured_attachments.each do |fa| %>
-  <% return if !@content_item.attachment_details(fa).present? %>
-  <% if @content_item.attachment_details(fa)['title'] == @content_item.title %>
-      <% content_for :title do %>
-        <%= t("content_item.schema_name.#{@content_item.document_type}.overview", count: 1) %>: <%= @content_item.title %>
-      <% end %>
-      <%
-         context_string =  t("content_item.schema_name.#{@content_item.document_type}.overview", count: 1) << ":"
-         context_inside = true
-      %>
-      <% break %>
-    <% end %>
-<% end %>
-
 <%= render 'govuk_publishing_components/components/title',
```

Comment:
I think we've lost something we want to keep here - line 9-11 adds both the document type and "overview" to the html title tag. While we're dropping the word overview, the title tag wants to line up with the H1 and include the document type. eg https://www.gov.uk/government/publications/national-probation-service-england-and-wales-divisions-map would want a title of:
Map: Probation Service England and Wales regional maps - GOV.UK